.travis.yml

```diff
@@ -17,14 +17,11 @@ addons:
     - python-dev
     - python-scipy
     - python-matplotlib
-    - texlive-latex-recommended
-    - texlive-fonts-recommended
-    - texlive-latex-extra
 
 matrix:
   include:
     - os: linux
-      compiler: gcc
+      compiler: clang
```

Comment:
It seems gcc is much slower and thus limit time is reached. As circle-ci relies on gcc, I moved to clang